src/dvclive/live.py

```diff
@@ -63,7 +63,7 @@ def __init__(
         resume: bool = False,
         report: Optional[str] = None,
         save_dvc_exp: bool = True,
-        dvcyaml: Union[str, bool] = True,
+        dvcyaml: Union[str, bool] = "dvc.yaml",
```

Comment:
Now that there is a constant value, it seems more transparent to make the default value a string. Keeping a boolean option for now to not break existing behavior.